core/modules/menu/menu.module

```diff
@@ -744,6 +744,11 @@ function menu_node_submit(Node $node, $form, $form_state) {
   if (!empty($form_state['values']['menu']['parent'])) {
     list($node->menu['menu_name'], $node->menu['plid']) = explode(':', $form_state['values']['menu']['parent']);
   }
+
+  // Replace tokens in the menu link title.
+  if (!empty($form_state['values']['menu']['link_title'])) {
+    $node->menu['link_title'] = token_replace($form_state['values']['menu']['link_title'], array('node' => $node));
+  }
```

Comment:
It seems we don't need this anymore, since that feature was removed...

Reply:
thank you!